digits/job.py

```diff
@@ -202,3 +211,44 @@ def disk_size_fmt(self):
         size = fs.get_tree_size(self._dir)
         return sizeof_fmt(size)
 
+    def get_tasks_recursively(self):
+        tasks = []
+        for task in self.tasks:
+            if task:
+                tasks.append(task)
+                tasks += task.get_tasks_recursively()
+        return tasks
```

Comment:
Wait, so if A is B's parent, then it is also in the job's list of tasks? That sounds like the node is in the graph twice.